tests/mpls/test_mpls.py

```diff
@@ -0,0 +1,594 @@
+import os
+import time
+import random
+import logging
+import pprint
+import json
+
+from abc import ABCMeta, abstractmethod
+
+import pytest
+
+import ptf.testutils as testutils
+import ptf.mask as mask
+import ptf.packet as packet
+
+from tests.common import reboot, port_toggle
+from tests.common.plugins.loganalyzer.loganalyzer import LogAnalyzer, LogAnalyzerError
+from tests.common.fixtures.duthost_utils import backup_and_restore_config_db_module
+from tests.common.config_reload import config_reload
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+#    pytest.mark.disable_loganalyzer,  # disable automatic loganalyzer
+    pytest.mark.topology('t1'),
+#    pytest.mark.sanity_check(skip_sanity=True)
+]
+BASE_DIR = os.path.dirname(os.path.realpath(__file__))
+DUT_TMP_DIR='/tmp'
+ADD_DIR = os.path.join(BASE_DIR, 'configs')
+
+LABEL_POP_ROUTES='label_pop_routes'
+LABEL_PUSH_ROUTES='label_push_routes'
+LABEL_SWAP_ROUTES='label_swap_routes'
+LABEL_DEL_ROUTES='label_del_routes'
+
+@pytest.fixture(scope='module')
+def setup(duthost, tbinfo, ptfadapter):
```

Comment:
Maybe we can move this fixture to the file `tests/mpls/conftest.py`?

Reply:
moved setup to conftest.py